.redocly.yaml

```diff
@@ -12,3 +12,9 @@ referenceDocs:
     colors:
       primary:
         main: "#32329f"
+  generateCodeSamples:
+    languages:  # Array of language config objects; indicates in which languages to generate code samples.
+      - lang: curl # Can be one of the following supported languages: "curl", "Node.js", "JavaScript".
```

Comment:
Will this work without the quotation marks?

Reply:
I don't think C# will work without quotes. It could use single quotes.

Reply:
Ah, I asked because curl is the only one in this example that doesn't have any quotes. So I wasn't sure if that's supported at all. We should probably use quotes with all languages consistently to avoid confusion.